test/cases/linux/logging/latest/custom_config/files/plugins/plugin.rb

```diff
@@ -0,0 +1,17 @@
+module Fluent
```

Comment:
If we are looking for a sample fluentd config for 3rd party app (In some cases we call it `plugin` but it's actually not very accurate), here is an example: https://github.com/GoogleCloudPlatform/fluentd-catch-all-config/blob/master/configs/config.d/apache.conf

Reply:
Understood, I have implemented the config used here https://github.com/GoogleCloudPlatform/google-cloud-ops-agents-ansible/blob/master/molecule/resources/example_configs/plugins/custom_config.conf